v2/pkg/engine/resolve/loader.go

```diff
@@ -679,9 +682,12 @@ func (l *Loader) appendSubgraphError(res *result, fetchItem *FetchItem, value *a
 func (l *Loader) mergeErrors(res *result, fetchItem *FetchItem, value *astjson.Value, values []*astjson.Value) error {
 	l.optionallyOmitErrorLocations(values)
 	l.optionallyRewriteErrorPaths(fetchItem, values)
-	l.optionallyAllowCustomExtensionProperties(values)
 	l.optionallyEnsureExtensionErrorCode(values)
 
+	if !l.allowAllErrorExtensionFields {
```

Comment:
should this be a default setting for pass through mode? it doesnt look like it is connected with passthrough currently